### ISSUE TITLE
Add a UI for Supplemental CSS

### ISSUE THREAD

**Issue by stpaultim** [backdrop/backdrop-issues#6786]
## Description of the need

Supplemental CSS is very close to getting into 1.30, despite an ongoing discussion about whether or not we should have a UI for this feature in Basis. We decided to merge the issue without a UI, holding out the option that we may YET incorproate a UI in 1.30 or later. 
https://github.com/backdrop/backdrop-issues/issues/4782

If we do not get a UI into 1.30, we would at least make sure to release a contrib module that provides a UI. 

## Proposed solution

Here is a screenshot the UI we were working on, before removing it from core.

![image](https://github.com/user-attachments/assets/1718b7de-29ff-4244-a80f-62e38d4f99f6)

**Comment by indigoxela** [comment]
My 2c to the UI-or-not discussion: People may wonder, why one of their sites looks differently to another of their sites, although they're using the same modules and theme ... They'd *never* figure out, it's because of the install time of the site.

To have the ability to fix that WTF, a UI for toggling on/off would be extremely helpful. Otherwise it's just a riddle for site builders and even if they figure out, only solvable by directly fiddling in a json file...

**Comment by stpaultim** [comment]
I have created a contrib module to add this UI as a temporary measure. I'm on the fence regarding the need for a UI for this feature and could go either way. 

https://github.com/backdrop-contrib/supplemental_css_ui

I think the presence of this UI is as likely to create confusion as the absense of the UI. But, in either case, I think we can mitigate that confusion.

**Comment by avpaderno** [comment]
I agree it would be difficult to understand why two different sites look different without a setting page.  
The only alternative I could see is an entry in the *Status report* page which could make clear what is causing the different look. (I am not saying this is the solution, nor could I suggest what that entry should say to make that clear.)

**Comment by indigoxela** [comment]
> The only alternative I could see is an entry in the Status report page which could make clear what is causing the different look.

I see some additional benefit in that info. And it shouldn't only be on the status page, but also on the debug page (admin/reports/debug).

But that's actually not topic of this issue here - should I create a new issue for that?

**Comment by avpaderno** [comment]
> But that's actually not topic of this issue here - should I create a new issue for that?

Adding information on /admin/reports/status and /admin/reports/debug can be done even if the settings page described here is added. I would create a new issue for that.
